src/main/resources/portal.properties.EXAMPLE

```diff
@@ -279,3 +279,9 @@ matchminer.token=
 # see: https://github.com/vy/hrrs/blob/master/README.md
 #hrrs.logging.filepath=
 #hrrs.enable.logging=false
+
+# Enable/disable Persistent Cache (true, false)
+# persistent_cache=true
+
+# Limit the size of gene queries (number). gene count * sample count < query_product_limit
+query_product_limit=1000000
```

Comment:
is this the current default? Should this only be for the public portal?

Reply:
This is the current default in the front end. I don't know if this should be public portal only. What determines whether or not to make a config value public portal only, and how does that change what you put in portal.properties?